driver/node.go

```diff
@@ -376,12 +376,13 @@ func (s *nodeService) nodeUnpublishFilesystemVolume(req *csi.NodeUnpublishVolume
 		if err := s.mounter.Unmount(target); err != nil {
 			return nil, status.Errorf(codes.Internal, "unmount failed for %s: error=%v", target, err)
 		}
-		if err := os.RemoveAll(target); err != nil {
-			return nil, status.Errorf(codes.Internal, "remove dir failed for %s: error=%v", target, err)
-		}
-		if err := os.Remove(device); err != nil {
-			return nil, status.Errorf(codes.Internal, "remove device failed for %s: error=%v", device, err)
-		}
+	}
+
+	if err := os.RemoveAll(target); err != nil {
+		return nil, status.Errorf(codes.Internal, "remove dir failed for %s: error=%v", target, err)
+	}
+	if err := os.Remove(device); err != nil {
```

Comment:
It seems that `os.Remove` returns not found error but `os.RemoveAll` does not.
So please check if the err is not found.

Reply:
My test code:
```go
package main
import "os"
import "log"
func main() {
        e := os.Remove("not_found")
        if e != nil {
                log.Print("remove error:", e)
        }
        e = os.RemoveAll("not_found")
        if e != nil {
                log.Print("removeall error:", e)
        }
}
```
The result:
```
2021/03/03 02:30:28 remove error:remove not_found: no such file or directory
```

Reply:
@toshipp Sorry...should I fix it like following?

```go
err = os.Remove(device)
if !os.IsNotExist(err) {
	return nil, status.Errorf(codes.Internal, "remove device failed for %s: error=%v", device, err)
}
```

ref: https://play.golang.org/p/XB9fTyhAgGe

Reply:
Yes.

Reply:
@toshipp Thank you. I fixed it.